CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ---
 
+## Master 
+
+* Update UIFont example on readme to match default template
```

Comment:
Please add a full-stop + 2 spaces at the end of this line. That's the Markdown syntax so that the next info (your name + PR link) goes to next line in same paragraph.